config/locales/titles/en.yml

```diff
@@ -29,6 +29,19 @@ en:
       information_not_verified: Personal information not verified
       phone_verification: Phone number not verified
     forget_all_browsers: Forget all browsers
+    icons:
```

Comment:
I think this `titles` file is intended for page `<title>...</title>` titles, not for labels like this.

Reply:
Perhaps we should create a general `labels` locale? I'm not sure where else it would go. Some of these files are based on specific views in which they appear, others seem to be more generalized

Reply:
Actually looks like `image_description` might be the place to put this